streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/runner/EnvInitializer.java

```diff
@@ -96,7 +97,7 @@ private void initInternalConfig(Environment springEnv) {
             .filter(springEnv::containsProperty)
             .forEach(key -> {
                 InternalOption config = InternalConfigHolder.getConfig(key);
-                assert config != null;
+                Preconditions.checkState(config != null);
```

Comment:
use `AssertUtils.state(...)` replace `Preconditions.checkState`, We should try to avoid use flink methods in the streampark-console, because streampark-console need to support multiple versions of flink, It is easy to cause conflicts when use flink methods

Reply:
Thanks for your review, updated.